src/WorldMap.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "TileLayer.h"
+#include "TileSheetDetails.h"
+#include <vector>
+#include <SFML\Graphics.hpp>
+#include <string>
+
+class WorldMap
+{
+public:
+	WorldMap();
+	WorldMap(const WorldMap&) = delete;
+	WorldMap& operator=(const WorldMap&) = delete;
+
+	void assignTileLayer(const std::vector<std::vector<int>>& tileData, const sf::Vector2i& mapSize, std::string&& tileLayerName);
```

Comment:
Since this is only used during 'initialization', it shouldn't be a public method.  Furthermore, we should do level parsing in the construction of this class.